app/src/main/java/com/lastaoutdoor/lasta/ui/components/LoadingAnim.kt

```diff
@@ -0,0 +1,18 @@
+package com.lastaoutdoor.lasta.ui.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun LoadingAnim(width: Int, tag: String = "LoadingBar") {

```

Comment:
The test tag is different since it is displayed on different screens and the tags need to be unique in the case of and E2E test. Also the width param is the one for the circular progress indicator and that can be different for instance on the loading screen it is quite large but on the discover screen can be smaller to emphasise that its loading a subset of the app's content.